Detectors/ITSMFT/MFT/calibration/src/NoiseCalibratorSpec.cxx

```diff
@@ -0,0 +1,181 @@
+  }
+  long tend = mEnd;
+  if (tend == -1) {
+    constexpr long SECONDSPERYEAR = 365 * 24 * 60 * 60;
+    tend = o2::ccdb::getFutureTimestamp(SECONDSPERYEAR);
+  }
+
+  auto toKeyValPairs = [](std::vector<std::string> const& tokens) {
+    std::vector<std::pair<std::string, std::string>> pairs;
+    Str strutils;
+    for (auto& token : tokens) {
+      //      auto keyval = splitString(token, '=');
+      auto keyval = Str::tokenize(token, '=', false);
+      if (keyval.size() != 2) {
+        // LOG(FATAL) << "Illegal command-line key/value string: " << token;
+        continue;
+      }
+
+      strutils.trim(keyval[1]);
+      std::pair<std::string, std::string> pair = std::make_pair(keyval[0], keyval[1]);
+      pairs.push_back(pair);
+    }
+
+    return pairs;
+  };
+  std::map<std::string, std::string> meta;
+  //  auto keyvalues = toKeyValPairs(splitString(mMeta, ';', true));
+  auto keyvalues = toKeyValPairs(Str::tokenize(mMeta, ';', true));
+
+  // fill meta map
+  for (auto& p : keyvalues) {
+    meta[p.first] = p.second;
+  }
+
+  long startTF, endTF;
+
+  const auto& payload = mCalibrator->getNoiseMap();
+
+  o2::ccdb::CcdbObjectInfo info(mPath, "NoiseMap", "noise.root", meta, tstart, tend);
+  auto flName = o2::ccdb::CcdbApi::generateFileName("noise");
+  auto image = o2::ccdb::CcdbApi::createObjectImage(&payload, &info);
+  info.setFileName(flName);
+  LOG(INFO) << "Sending object " << info.getPath() << "/" << info.getFileName()
+            << " of size " << image->size()
+            << " bytes, valid for " << info.getStartValidityTimestamp()
+            << " : " << info.getEndValidityTimestamp();
+
+  using clbUtils = o2::calibration::Utils;
+  output.snapshot(
+    Output{clbUtils::gDataOriginCLB, clbUtils::gDataDescriptionCLBPayload, 0}, *image.get());
+  output.snapshot(
+    Output{clbUtils::gDataOriginCLB, clbUtils::gDataDescriptionCLBInfo, 0}, info);
+}
+
+void NoiseCalibratorSpec::endOfStream(o2::framework::EndOfStreamContext& ec)
+{
+  sendOutput(ec.outputs());
+}
+
+DataProcessorSpec getNoiseCalibratorSpec(bool useDigits)
+{
+  o2::header::DataOrigin detOrig = o2::header::gDataOriginMFT;
+  std::vector<InputSpec> inputs;
+  if (useDigits) {
+    inputs.emplace_back("digits", detOrig, "DIGITS", 0, Lifetime::Timeframe);
+    inputs.emplace_back("digitsROF", detOrig, "DIGITSROF", 0, Lifetime::Timeframe);
+  } else {
+    inputs.emplace_back("compClusters", detOrig, "COMPCLUSTERS", 0, Lifetime::Timeframe);
+    inputs.emplace_back("patterns", detOrig, "PATTERNS", 0, Lifetime::Timeframe);
+    inputs.emplace_back("ROframes", detOrig, "CLUSTERSROF", 0, Lifetime::Timeframe);
+  }
+
+  using clbUtils = o2::calibration::Utils;
+  std::vector<OutputSpec> outputs;
+  outputs.emplace_back(
+    ConcreteDataTypeMatcher{clbUtils::gDataOriginCLB, clbUtils::gDataDescriptionCLBPayload});
+  outputs.emplace_back(
+    ConcreteDataTypeMatcher{clbUtils::gDataOriginCLB, clbUtils::gDataDescriptionCLBInfo});
```

Comment:
Respectively, in the getNoiseCalibratorSpec the outputs should be declared as 
````
outputs.emplace_back(ConcreteDataTypeMatcher{clbUtils::gDataOriginCDBPayload, "MFT_NoiseMap"});
outputs.emplace_back(ConcreteDataTypeMatcher{clbUtils::gDataOriginCDBWrapper, "MFT_NoiseMap"});
````